src/terra-dev-site/test/application-navigation/ApplicationNavigationDisabledPromptsTest.test.jsx

```diff
@@ -48,7 +50,7 @@ const ApplicationNavigationDisabledPromptsTest = () => {
   }];
 
   return (
-    <ApplicationBase locale="en">
+    <ApplicationBase locale={applicationIntl.locale}>
```

Comment:
I think this is only working out because we are rendered in terra-dev-site and the local is being provided by the root Application Base. That being said, I think this is fine since this is a test example.